pymatgen/analysis/chemenv/coordination_environments/chemenv_strategies.py

```diff
@@ -498,10 +528,10 @@ class SimplestChemenvStrategy(AbstractChemenvStrategy):
     )
 
     STRATEGY_DESCRIPTION = (
-        "    Simplest ChemenvStrategy using fixed angle and distance parameters \n"
-        "    for the definition of neighbors in the Voronoi approach. \n"
-        "    The coordination environment is then given as the one with the \n"
-        "    lowest continuous symmetry measure."
+        "Simplest ChemenvStrategy using fixed angle and distance parameters \n"
+        "for the definition of neighbors in the Voronoi approach. \n"
+        "The coordination environment is then given as the one with the \n"
+        "lowest continuous symmetry measure."
```

Comment:
Do want have a good reason to indent this description? The unit test seems to be expecting indentation, if not I would remove the indentation from unit test as well.

Reply:
@davidwaroquiers ?

Reply:
Fine with me to remove indentation.